domain-models-runtime-it/src/test/java/org/folio/rest/ApiTestBase.java

```diff
@@ -67,13 +70,16 @@ static void beforeAll() {
         contentType(ContentType.JSON);
 
     // delete tenant (schema, tables, ...) if it exists from previous tests, ignore errors.
```

Comment:
```suggestion
    // delete tenant (schema, tables, ...) if it exists from previous tests.
```
to match the code that asserts status code 204.

Reply:
thx. Will update.